reporter-web-app/src/components/PackageFindingsTable.js

```diff
@@ -23,128 +23,111 @@ import { Table } from 'antd';
 import { CopyrightOutlined, FileTextOutlined } from '@ant-design/icons';
 
 // Generates the HTML to display scanFindings as a Table
```

Comment:
commit-msg:

previous showed package's scan results. -> previously shown package.
has now been fixed -> is now fixed

Reply:
fixed